examples/bridge-app/linux/main.cpp

```diff
@@ -69,6 +69,12 @@ void emberAfPostAttributeChangeCallback(EndpointId endpoint, ClusterId clusterId
     }
 }
 
+bool emberAfBasicClusterMfgSpecificPingCallback(void)
+{
+    emberAfSendDefaultResponse(emberAfCurrentCommand(), EMBER_ZCL_STATUS_SUCCESS);
+    return true;
+}
+
```

Comment:
I have added `MfgSpecificPing` command mostly to be able our old `echo` mechanism and in order to check that manufacturer specific methods on a cluster are generated properly. If you don't need it you can just disable the `MfgSpecificPing` method from the basic cluster in the ZAP tool.

It depends if you really want it or not. For the record this method is just a test to send a message to the device, and it will reply back with a default response that contains `EMBER_ZCL_STATUS_SUCCESS`.

Reply:
Done